docs/articles/datadog-junit.mdx

```diff
@@ -0,0 +1,120 @@
+# Datadog
+
+Datadog’s CI / Test Visibility product can ingest JUnit XML reports so you can visualize test performance and flake rate straight from your CI/CD pipelines. However, if you’re trying to [decouple test execution from CI/CD automation](https://testkube.io/learn/stop-running-tests-with-your-ci-cd-tool), you may want to send JUnit reports to Data Dog directly from a Testkube TestWorkflow execution. This guide shows how to upload those reports immediately after a TestWorkflow finishes by using a reusable **TestWorkflowTemplate**.
+
+## How it works
+
+1. Your test step produces JUnit XML files and saves them under **/data/artifacts** (the default working directory in Testkube TestWorkflow).
+2. A template step runs the official `datadog/ci` container and calls `datadog-ci junit upload`, pointing it at the artifacts directory and passing metadata such as the service name.
+3. Any TestWorkflow can reference the template with a single `template:` step, keeping your pipelines clean and DRY.
+
+---
+
+## Prerequisites
+
+- Testkube control plane deployed in your cluster
+- A Datadog account with **CI or Test Visibility** enabled
+- A Kubernetes Secret that stores your **Datadog API key**:
+
+```bash
+kubectl create secret generic datadog-api-key \
+  --from-literal=DATADOG_API_KEY=<your-key> \
+  -n testkube
+
+```
+
+---
+
+## 1. Create the `TestWorkflowTemplate`
+
+```yaml
+kind: TestWorkflowTemplate
+apiVersion: testworkflows.testkube.io/v1
+metadata:
+  name: datadog-ci-junit-upload
+  namespace: testkube
+spec:
+  # Optional parameter you can override from the TestWorkflow
+  config:
+    service:
+      description: Name of the service or library being tested
```

Comment:
perhaps clarify that this is metadata used by datadog to categorise results!? (if that's the case..)